django/db/models/functions/__init__.py

```diff
@@ -4,6 +4,7 @@
 from django.db.models import (
     DateTimeField, Func, IntegerField, Transform, Value,
 )
+from django.db.models.functions.date import *  # NOQA
```

Comment:
I'd rather not use `import *` since it prevents detecting unused imports. Alternatively, you could move functions.py -> base.py or something and make ` __init__.py` only contains imports.